cmake/pkgconfig/ignition.in

```diff
@@ -1,10 +1,10 @@
-prefix=${pcfiledir}/../..
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+prefix=${pcfiledir}/@PC_CONFIG_RELATIVE_PATH_TO_PREFIX@
+libdir=${prefix}/@LIB_INSTALL_DIR@
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
```

Comment:
we could just delete these 3 lines I suppose

Reply:
+1, maybe we can just drop a comment about the lack of headers/libs so anyone looking into the generated file don't panic about the lack of data.

Reply:
comment added in 2a90e1701b193f79a5ca17e8f88ab345b985dcbc, unneeded lines removed in c20c209d960742e0c71c918cd636de8a09caeb37